test/functional/p2p_filter.py

```diff
@@ -67,7 +70,13 @@ def run_test(self):
 
         self.log.info('Check that too large filter is rejected')
         with self.nodes[0].assert_debug_log(['Misbehaving']):
-            filter_node.send_and_ping(msg_filterload(data=b'\xaa', nHashFuncs=51, nTweak=0, nFlags=1))
+            filter_node.send_and_ping(msg_filterload(data=b'\xaa', nHashFuncs=MAX_BLOOM_HASH_FUNCS+1))
+        with self.nodes[0].assert_debug_log(['Misbehaving']):
+            filter_node.send_and_ping(msg_filterload(data=b'\xbb'*(MAX_BLOOM_FILTER_SIZE+1)))
+
+        self.log.info('Check that too large data element to add to the filter is rejected')
+        with self.nodes[0].assert_debug_log(['Misbehaving']):
+            filter_node.send_and_ping(msg_filteradd(data=b'\xcc'*(MAX_SCRIPT_ELEMENT_SIZE+1)))
```

Comment:
I think you need to successfully load a filter first to test this condition?

Reply:
Good point!  Sorry I missed this when reviewing.  I added this in #18726

Reply:
@MarcoFalke: You are right, the misbehaviour is caused for another reasons here, the test is basically equivalent to "'Check that sending "filteradd" if no filter is set is treated as misbehavior'" below.
@robot-visions: Thanks for adding the fix!